src/chapters/09/main.tex

```diff
@@ -658,12 +649,75 @@ \section{Solving with R}\label{sec:solving-with-R}
 print(solution_with_swap_stops)
 \end{Rin}
 
-giving:
+gives:
 
 \begin{Rout}
  [1]  0 11  4 10  6  2  7  8  5  3  1  9 12  0
 \end{Rout}
 
+This has a cost:
+
+\begin{Rin}
+cost <- get_cost(
+    tour = solution_with_swap_stops,
+    distance_matrix = distance_matrix
+)
+print(cost)
+\end{Rin}
+
+which gives:
+
+\begin{Rout}
+[1] 373
+\end{Rout}
+
+Therefore, using this particular algorithm, a pretty good route it found, with a
+total distance of 373.
+
+It is important to note that this may not be the optimal route, and tweaking the
+algorithm slightly may produce better solutions.
+For example, one way to do this is to use a different neighbourhood operator.
+One example is instead of swapping two stop, revering the path between those two
+stops. The \mintinline{R}{reverse_path} function does this:
```

Comment:
```suggestion
It is important to note that this may not be the optimal route, and different algorithms
 may produce better solutions.
For example, one way to modify the algorithm is to use a different neighbourhood operator.
Instead of swapping two stops, reverse the path between those two
stops. The \mintinline{R}{reverse_path} function does this:
```